src/tests/getOneAccom.test.js

```diff
@@ -0,0 +1,44 @@
+import Chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+Chai.use(chaiHttp);
+Chai.should();
+
+describe('Barefoot nomad should allow multi city trips', () => {
+  it('It should login successfuly', (done) => {
+    Chai
+      .request(app)
+      .post('/api/v1/auth/signin')
+      .send({
+        email: 'blaisefr@gmail.com',
+        password: 'password',
+      })
+      .end((err, res) => {
+        res.body.should.have.status(200);
+        done();
+      });
+  });
+
+  it('It should get one accommodation with the given id', (done) => {
+    Chai
+      .request(app)
+      .get('/api/v1/accommodations/c8e9428a-6d60-4083-8ce3-334c62afe72c')
+      .end((err, res) => {
+        res.body.should.have.status(200);
+        res.body.should.have.property('message', 'Accommodation retrieved successfully');
+        done();
+      });
+  });
+
+  it('It should not get one accommodation if the given id don\'t exist', (done) => {
```

Comment:
```suggestion
  it('should not get one accommodation if the given id don\'t exist', (done) => {
```
@AbdoulNuru Get a rid of one ``it``